### ISSUE TITLE
MeteorToys prevents running in production, if using Notifications

### ISSUE THREAD

**Issue by 1jerry** [MeteorToys/allthings#101]
I cannot run my app in production or with --production when meteortoys:notifications are installed.  Notifications are still documented as current, but in the browser it references`Package[meteortoys:toykit']` as if it exists and prevents running in production mode because the client crashes while loading packages.

Meteor 1.4.2.3
Windows 10 (dev & --production test)
some *NIX version for production server
meteortoys:allthing  3.0.0
meteortoys:notifications 3.0.0

**Comment by msavin** [comment]
This sounds very strange - as you can see, the package is really simple: 
https://github.com/MeteorToys/notifications/blob/master/client/main.js

If you run `Package["meteortoys:toykit"]` in the console on a --production build, what does it return?

**Comment by 1jerry** [comment]
I created a default project with no template.  I then added `allthings` and `notifications`.  (`notifications` now shows version 4.0.0, which is strange because the package.js in your repo still shows 2.1.0).

"toykit" does not exist in Package, so it is `undefined` in all cases with --production.

If I remove `notifications`, everything runs fine.  If I include the `notifications` package, I get the error **Uncaught TypeError: Cannot read property 'Note' of undefined** in the client console (Chrome 58) and the app does not run.  The error occurs on this line starting:

```
!function(){var e=Package.meteor.Meteor,o=Package.meteor.global,t=Package.meteor.meteorEnv,a;(function(){a=function(e,o){console.log(e),o&&console.log(o)},Package["meteortoys:toykit"].Note&&(a=Package["meteortoys:toykit"].Note)})....
```
and ending with:
```
(Package["meteortoys:notifications"] = {}, {
        Note: a
    })
```

Ahhhhhhhhhhhhh.  Of course.  Your code in notifications/client/main.js is missing the bold code:

if (**_Package["meteortoys:toykit"] &&_** Package["meteortoys:toykit"].Note) {

(I see why it hasn't been touched in two years.  1) it's leveraging existing functions and 2) I seem to be the _only person_ who has used meteortoys:notifications in an production app.
